backend/src/server.js

```diff
@@ -1,6 +1,8 @@
 require('express-async-errors');
 const logger = require('./winston.js');
-const config = require('../../config.json');
+const fs = require('fs');
+const config = require('../../config/config.json');
+const api = require('../../config/api.json');
```

Comment:
hmm.. those configs are basically constants, I think it would be appropriate to use the same naming style as for error codes etc, what do you think?